CONTRIBUTING.md

```diff
@@ -185,6 +189,9 @@ To learn about all possible rules you can execute, run:
 python x.py build --help --verbose
 ```
 
+Note: Previously `./configure` and `make` were used to build this project,
+but these are now deprecated.
```

Comment:
"Deprecated" is a little strong here I think because we don't plan on ever removing them. Moreso we just recommend "x.py" for a better build experience nowadays.